src/main/webapp/app/code-editor/ace/code-editor-ace.component.ts

```diff
@@ -81,14 +112,118 @@ export class CodeEditorAceComponent implements OnInit, AfterViewInit, OnChanges
      * @param {SimpleChanges} changes
```

Comment:
Could you extend the documentation here and describe what kind of changes you do with the annotations?

Reply:
I will add a more explanatory comment to this section.

Reply:
Done.